companies/swapfiets-it.json

```diff
@@ -0,0 +1,18 @@
+{
+    "slug": "swapfiets-it",
+    "relevant-countries": [
+        "it"
+    ],
+    "name": "Swapfiets Italia s.r.l.",
+    "address": "Via Adda 16\n20090 Opera MI\nItaly",
+    "phone": "+39 02 8295 8065",
+    "email": "privacy@swapfiets.com",
+    "web": "https://swapfiets.it/",
+    "sources": [
+        "https://swapfiets.it/en-IT/privacy",
```

Comment:
This source lists "Swapfiets B.V" as the responsible entity.

Reply:
But as the phone number and address are in Italy-specific, we can't really merge this. @baltpeter What do you think?

Reply:
Haven't check the source but I agree. In that case, they should stay separate records. We can have multiple records with the same `name`.

Reply:
I think this is a translation/copy-paste error. The [Italian version](https://swapfiets.it/it-IT/privacy) lists Swapfiets Italia

Reply:
That seems likely, good catch! I would prefer to use the Italian source then.

Reply:
👍 changed it